package.json

```diff
@@ -239,7 +240,7 @@
     "test-server:coverage": "npm run -s test-server -- --coverage",
     "test-server:watch": "npm run -s test-server -- --watch",
     "translate": "i18n-calypso --format pot --output-file ./calypso-strings.pot -e date \"**/*.js\" \"**/*.jsx\" \"!build/**\" \"!node_modules/**\" \"!public/**\" \"!client/extensions/**\"",
-    "update-deps": "npm run -s rm -- node_modules && npm run -s rm -- npm-shrinkwrap.json && npm install && npm install && npm shrinkwrap --dev"
+    "update-deps": "npm run -s rm -- node_modules && npm run -s rm -- npm-shrinkwrap.json && npm install && npm shrinkwrap --dev"
```

Comment:
This is not a bug, sometimes you have to run `npm install` twice to ensure all the dependencies have been installed. See https://github.com/npm/npm/issues/10727

Reply:
Wow, I didn't know that. Still I would call that a bug :).

I'll revert that specific change.